l10n_it_sdi_channel/models/sdi.py

```diff
@@ -168,3 +171,230 @@ def receive_fe(
             if e_invoice_user:
                 attachment_model = attachment_model.sudo(e_invoice_user.id)
         return attachment_model.create(all_attachments_values)
+
+    @api.model
+    def _search_attachment_out_by_notification(
```

Comment:
Questo è specifico per la pec?

Reply:
No, le notifiche arrivano a qualsiasi canale che comunichi con SdI e hanno sempre lo stesso formato, regolato dallo schema https://www.fatturapa.gov.it/export/documenti/messaggi/v1.1/MessaggiTypes_v1.1.xsd.
Puoi trovare il resto delle specifiche generiche in https://www.fatturapa.gov.it/it/norme-e-regole/DocumentazioneSDI/.